text/3371-cargo-target-directories.md

```diff
@@ -240,10 +272,17 @@ Note that while precedent set by other languages is some motivation, it does not
 Please also take into consideration that rust sometimes intentionally diverges from common language features.
 -->
 
-- [`targo`](https://github.com/sunshowers/targo) by @sunshowers
-- [rust-lang/cargo#11156](https://github.com/rust-lang/cargo/issues/11156)
+## `bazel`
+
+The [`bazel`] build system has a similar feature called the [`outputRoot`](https://docs.bazel.build/versions/5.4.0/output_directories.html), which is always active and has default directories on all major platforms (Linux, macOS, Windows).
+
+The naming scheme is as follow: `<outputRoot>/_bazel_$USER/` is the `outputUserRoot`, used for all builds done by `$USER`. Below that, projects are identified by the MD5 hash of the path name of the workspace directory (computed after resolving symlinks).
+
+The `outputRoot` can be overridden using `--output_base=...` (this is `$CARGO_TARGET_DIRECTORIES`, the subject of this RFC) and the `outputUserRoot` with `--output_user_root=...` (this is close to using `$CARGO_TARGET_DIR`, already possible in today's `cargo`).
```

Comment:
Is `--output_user_root` `<outputRoot>/_bazel_$USER/$MD5` (implied by the text) or `<outputRoot>/_bazel_$USER` (implied by the name)?

Reply:
I have no idea sorry